pkg/fork-tests/tests/20221114-yearn-linear-pool/readme.md

```diff
@@ -0,0 +1,12 @@
+# 2022-11-14 - Yearn Linear Pool
+
+First deployment of the `YearnLinearPoolFactory`, for Linear Pools with a Yearn yield-bearing token.
+Already fixes the reentrancy issue described in https://forum.balancer.fi/t/reentrancy-vulnerability-scope-expanded/4345.
+Also has a fix in the `YearnLinearPoolRebalancer` to handle tokens which require the `SafeERC20` library for approvals.
+
+## Useful Files
+
+- [`YearnLinearPoolFactory` artifact](./artifact/YearnLinearPoolFactory.json)
+- [`YearnLinearPool` artifact](./artifact/YearnLinearPool.json)
+- [`YearnLinearPoolRebalancer` artifact](./artifact/YearnLinearPoolRebalancer.json)
+- [`YearnShareValueHelper` artifact](./artifact/YearnShareValueHelper.json)
```

Comment:
We don't need this artifact because this contract never gets deployed on its own; it is inherited by the Pool.
```suggestion
```